patterns/header-large-dark.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Title: WooCommerce Large Header | Dark
+ * Slug: woocommerce-blocks/header-large-dark
+ * Categories: WooCommerce
+ */
+?>
+<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0px","padding":{"top":"1rem","right":"1rem","bottom":"1rem","left":"1rem"}},"elements":{"link":{"color":{"text":"var:preset|color|background"}}}},"backgroundColor":"black","textColor":"background","layout":{"type":"default"}} -->
+<div class="wp-block-group alignfull has-background-color has-black-background-color has-text-color has-background has-link-color" style="padding-top:1rem;padding-right:1rem;padding-bottom:1rem;padding-left:1rem"><!-- wp:group {"style":{"spacing":{"blockGap":"1rem"},"typography":{"fontSize":"13px"}},"layout":{"type":"flex","flexWrap":"wrap","justifyContent":"right"}} -->
+<div class="wp-block-group" style="font-size:13px"><!-- wp:search {"label":"Search","showLabel":false,"placeholder":"Search","width":220,"widthUnit":"px","buttonText":"Search","buttonPosition":"no-button","backgroundColor":"background"} /-->
+
+<!-- wp:woocommerce/mini-cart {"textColor":"background","style":{"typography":{"fontSize":"14px"}}} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"16px","padding":{"top":"1rem","left":"0px","bottom":"2rem"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"center"}} -->
+<div class="wp-block-group" style="padding-top:1rem;padding-bottom:2rem;padding-left:0px"><!-- wp:site-logo {"shouldSyncIcon":true} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:navigation {"ref":515,"textColor":"background","layout":{"type":"flex","justifyContent":"center"}} /--></div>
```

Comment:
I see the message as shown in the screenshot below for the `Large dark header` pattern:
![image](https://user-images.githubusercontent.com/16707866/194228745-8acbd426-e012-4fa8-83d9-d398a3c69d30.png)

I am not exactly sure, but this might be because of `"ref":515` attribute. After removing this attribute, it seems to work fine. 😯
@albarin Are you able to reproduce this issue?

Reply:
Hey, good catch! Yes, it's because of the `ref` attribute, I just pushed a fix, could you try again? 🙏